### ISSUE TITLE
Application crash due to the enhanced debug mode

### ISSUE THREAD

**Issue by jsproede** [dieuhd/sonar-quality-gate#54]
Hi :)

With pull request #53 you've introduced an enhanced debug mode which is enabled in our pipelines by default to see a more verbose log.
Since the update a few hours ago, we've issues with the changes you made for the enhanced debug mode:

```
{"action":"generate report","data":{"provide":"gitlab"}}
{"action":"sonar get task status","data":"/api/ce/activity"}
{"action":"[axios] request","data":{"transitional":{"silentJSONParsing":true,"forcedJSONParsing":true,"clarifyTimeoutError":false},"transformRequest":[null],"transformResponse":[null],"timeout":0,"xsrfCookieName":"XSRF-TOKEN","xsrfHeaderName":"X-XSRF-TOKEN","maxContentLength":-1,"maxBodyLength":-1,"headers":{"common":{"Accept":"application/json, text/plain, */*"},"delete":{},"get":{},"head":{},"post":{"Content-Type":"application/x-www-form-urlencoded"},"put":{"Content-Type":"application/x-www-form-urlencoded"},"patch":{"Content-Type":"application/x-www-form-urlencoded"},"Authorization":"Bearer [MASKED]"},"method":"get","url":"[REDACTED]/api/ce/activity?component=[REDACTED]&onlyCurrents=true"}}
TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'ClientRequest'
    |     property 'socket' -> object with constructor 'TLSSocket'
    --- property '_httpMessage' closes the circle
    at JSON.stringify (<anonymous>)
    at Logger.debug (/usr/local/lib/node_modules/sonar-quality-gate/build/utils/log.js:18:26)
    at onResponse (/usr/local/lib/node_modules/sonar-quality-gate/build/http/interceptors.js:14:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

After the error the whole application crashes and doesn't continue decorating the merge requests in GitLab.

**Comment by dieuhd** [comment]
@jsproede 
i have pushed code for fix this bug #56 , please check it again.

**Comment by jsproede** [comment]
Hi @dieuhd,

sorry for the delay. The issue has been fixed with the changes you've pushed. Thank you! :)
